### PR TITLE
Data block

### PR DESCRIPTION
**Pull request by gaspard**
# WIP

- [x] server tests
- [x] data tests
- [x] server runs with node_modules installed in monorepo

## After this PR, next steps are:

- [ ] server subscriptions
- [ ] client subscriptions
- [ ] client offline
- [ ] client sync

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,10 +1,8 @@
-**/build
```
Comment (henri-hulski) [inline .prettierignore]: I think we should also remove it from `.eslintignore` and maybe more ignore files.

File: packages/apps/todomvc-server/src/controller/TodoController.ts [hunk 2]
```diff
@@ -10,17 +11,19 @@ export class TodoController {
 
   @Mutation()
   async mutateTodo(args) {
+    const all = await Todo.find()
```
Comment (henri-hulski) [inline packages/apps/todomvc-server/src/controller/TodoController.ts]: Why you need this? To test if any Todo exists?

Comment (henri-hulski) [reply packages/apps/todomvc-server/src/controller/TodoController.ts]: Than I think you could remove `const all = ` as you don't use it afterwords. (linting error)

Comment (gaspard) [reply packages/apps/todomvc-server/src/controller/TodoController.ts]: Oups. Ammended the commit.

**Pull request merged**